deployment/arm/deploy-cosmosDB.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+cd "$parent_path"
+
+businessUnit=$1
+appName=$2
+env=$3
+primaryRegion=$4
+secondaryRegion=$5
+tertiaryRegion=$6
+
+scope="$businessUnit-$appName-$env-glb"
+resourceGroupName="rg-$scope"
+
+echo "Resource Group: $resourceGroupName"
+echo "Business Unit: $businessUnit"
+echo "App Name: $appName"
+echo "Environment: $env"
+
+echo "Creating global resource Group: $resourceGroupName"
+az group create \
+  --name $resourceGroupName \
+  --location centralus
```

Comment:
why central us? can we make this configurable with a default?  Would it make sense to default this to the group deployment's primary region?